Add more info about inbound filters (#9158)
getsentry/sentry-docs: docs/product/data-management-settings/filtering/index.mdx

If you had a legacy browser filter on before the update, the old filter will appear in your settings as "Deprecated". Deprecated legacy browser filters still work. However, if you turn them off, you won’t be able to turn them on again and will need to use the new filters instead.

### Browser Extension Errors

> The browser extension filter checks an event to see if it matches any known browser extension errors. The filter will check both for the error message to see if the error is a known to come from browser extensions and it will check the source of the event, to see if it is coming from a browser extension.

Reviewer: ```suggestion
Some browser extensions are known to cause errors. You can use the browser extension filter to filter these out. The filter checks the error message and the event source to see if it's a known error coming from a browser extension.
```

### Web Crawler Errors

> The web crawler filter allows you to filter out errors from web crawlers from the following sites: Baidu, Yahoo, Sogou, Facebook, Alexa, generic bots and spiders, Slack, Google Indexing, Pingdom, Lytics, AWS Security Scanner, Hubspot, and Bytedance. Errors from Slack's Slackbot web crawler will not be filtered. 

Reviewer: ```suggestion
Due to their nature, web crawlers often encounter errors that normal users won't see. You can use the web crawler filter to filter out errors from web crawlers from the following sites: Baidu, Yahoo, Sogou, Facebook, Alexa, Slack, Google Indexing, Pingdom, Lytics, AWS Security Scanner, Hubspot, Bytedance, and other generic bots and spiders. Errors from Slack's Slackbot web crawler will not be filtered. 
```

> Due to their nature, web crawlers often encounter errors that normal users won't see. You can use the web crawler filter to filter out errors from web crawlers from the following sites: Baidu, Yahoo, Sogou, Facebook, Alexa, Slack, Google Indexing, Pingdom, Lytics, AWS Security Scanner, Hubspot, Bytedance, and other generic bots and spiders. Errors from Slack's Slackbot web crawler will not be filtered. 

Reviewer 2: Is it worth mentioning that we use the user-agent to check these?

Author: yeah, let me add something in!
